src/locales/hu.yaml

```diff
@@ -613,6 +619,25 @@ app:
       screws_tilt_calculate: Screws_Tilt_Calculate
       bed_screws_adjust: BED_SCREWS_ADJUST
       force_move: FORCE_MOVE
+    label:
+      stats_active_extruder: >-
+        <span class="secondary--text">Az aktív extruder konfigurációja be van állítva</span>
+        %{filamentDiameter} mm átmérőjű filament
+        <span class="secondary--text">és</span>
+        %{nozzleDiameter} mm fúvóka
+      stats_volumetric_flow: >-
+        Extrudálás: %{extrudeSpeed} mm/s<span class="secondary--text">,
+        az extrudernek biztosítania kell a</span>t
+        becsült térfogatáram %{estimatedVolumetricFlow} mm³/s
+      stats_extruded_length: >-
+        A filament extrudálása %{extrudeLength} mm-nél
+        %{extrudeFactor} % folyamot<span class="secondary--text">,
+        biztosít egy</span>-t
+        becsült kimeneti hosszon %{estimatedExtrudedLength} mm
+      stats_max_speed: >-
+        <span class="secondary--text">Feltéve, hogy a</span>
+        %{layerHeight} mm réteg magassága<span class="secondary--text">,
+        a</span> becsült maximális nyomtatási sebesség %{estimatedMaxSpeed} mm/s
```

Comment:
Just wanted to double-check that these are appearing correctly:

![image](https://user-images.githubusercontent.com/85504/208269298-c8d70c88-d305-485d-8982-3375ce4e19a2.png)

I found it odd that some of the highlighted areas (marked by the `<span>` tabs above) are showing mid word, so wanted to check with you first

Reply:
I tried to translate correctly. The location of the words in this context is unusual, because I have not dealt with such a translation part before.